### DISCUSSION TITLE
Hacked account

### DISCUSSION THREAD

**Discussion by codemania126** [community/community#111239] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hello community members and moderators!
Recently my github account was stolen. The scammer changed handle and email so I couldn't track and created a new account as a last resort.
BUT today I found out that the scammer changed the handle as `forwardtoward` and also changed the email.
Please, help me recover my account!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @codemania126 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
